legal-api/tests/unit/services/test_involuntary_dissolution.py

```diff
@@ -39,15 +39,16 @@
 
 
 @pytest.mark.parametrize(
-        'test_name, eligible', [
-            ('TEST_ELIGIBLE', True),
-            ('TEST_INELIGIBLE', False)
+        'test_name, isAdminFreeze, eligible', [
+            ('TEST_ELIGIBLE', False, True),
+            ('TEST_INELIGIBLE', False, False),
+            ('TEST_INELIGIBLE_ADMIN_FREEZE', True, False)
         ]
 )
-def test_check_business_eligibility(session, test_name, eligible):
+def test_check_business_eligibility(session, test_name, isAdminFreeze, eligible):
```

Comment:
Can you revert the changes here and write separate tests for your work? This test I wrote is to check the general functionality of method `check_business_eligibility` instead of the underlying query. The query will be used in more than one method. It's better to write new & seperate tests to check whether frozen buisnesses are excluded.

You can use this [test](https://github.com/bcgov/lear/blob/main/legal-api/tests/unit/services/test_involuntary_dissolution.py#L118) as reference.

Reply:
Thanks for letting me know, going to work on it.

Reply:
created new unit tests to check query results excluded the business with admin_freeze is true, along testing eligibility returns the results as expected.